runsc/cmd/nvproxy/list_supported_drivers.go

```diff
@@ -0,0 +1,46 @@
+package nvproxy
```

Comment:
This needs the copyright that you have at the top of the runsc/cmd/nvproxy/nvproxy.go file.

Reply:
Added the copyright notice to this as well